content/cockpit-148.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Cockpit 148
+author: Martin Pitt
+date: 2017-08-04 12:00
+tags: cockpit, linux
+slug: cockpit-148
+category: release
+summary: Cockpit with Internet Explorer support
+comments: true
+---
+
+Cockpit is the [modern Linux admin interface](http://cockpit-project.org/). We release regularly.
+Here are the release notes from version 148.
+
+### Support Cockpit in Internet Explorer
+
+When using Microsoft Windows Internet Explorer, navigating between pages and
+various other operations caused lost state changes, flickering, and lots of
+JavaScript errors. These have been corrected, and automatic tests now run with
+Internet Explorer as well.
+
+### Update Cluster/Registry design for image streams
+
+The design of the image streams on the Cluster and Registry pages has been
+updated to match the design of the current OpenShift Web Console.
+
+![Image Stream Tag Design](images/registry-imagestreams-design.png)
+
+### Delete Kubernetes session tokens when logging out of Cockpit
```

Comment:
This doesn't apply to Cockpit. It only applies to the "Registry Console" so

"Delete Kubernetes session tokens when logging out of the Registry Console or Dashboard"